The initrd refuses to work for me without the attached patch
(actually, initrd works, but nothing else does: console is hosed).
I did not see anything on the list. Am I the only one who
uses initrd?

-- Pete

--- linux-2.4.21-pre5/init/do_mounts.c	2003-03-12 21:21:05.000000000 -0800
+++ linux-2.4.21-pre5-nip/init/do_mounts.c	2003-03-12 21:43:03.000000000 -0800
@@ -813,6 +813,8 @@
 	sys_fchdir(root_fd);
 	sys_chroot(".");
 	sys_umount("/old/dev", 0);
+	close(old_fd);
+	close(root_fd);
 
 	if (real_root_dev == ram0) {
 		sys_chdir("/old");
-
To unsubscribe from this list: send the line "unsubscribe linux-kernel" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html
Please read the FAQ at  http://www.tux.org/lkml/
